packages/rest-api-client/src/client/types/app/notification.ts

```diff
@@ -33,3 +33,20 @@ export type PerRecordNotificationForResponse = {
   }>;
   revision: string;
 };
+
+export type ReminderNotificationForResponse = {
+  timing: {
+    code: string;
+    daysLater: number | string;
+    hoursLater: number | string;
```

Comment:
```suggestion
    daysLater: string;
    hoursLater: string;
```

Since this is the type for **response**, I thought the returned values are always string, right?